apps/marketplace/components/Brief/BriefDownloadDocuments.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import LoadingIndicatorFullPage from 'shared/LoadingIndicatorFullPage/LoadingIndicatorFullPage'
+import styles from './BriefDownloadDocuments.scss'
+
+const canDownload = () => {
+  const hasDownload = typeof document.createElement('a').download !== 'undefined'
+  const hasObjectURL = typeof window.URL !== 'undefined' && window.URL.createObjectURL !== 'undefined'
+  return hasDownload && hasObjectURL
+}
+
+export class BriefDownloadDocuments extends Component {
+  static defaultProps = {
+    briefDocumentsData: null
+  }
+
+  componentDidMount() {
+    this.props.downloadBriefDocuments(this.props.brief.id)
+  }
+
+  createDownloadElement() {
+    if (canDownload()) {
+      return (
+        <a
+          className={styles.hidden}
+          href={window.URL.createObjectURL(this.props.briefDocumentsData)}
```

Comment:
Also use match not window. Anti react pattern

Reply:
Same as above.